x-pack/plugins/alerting/public/alert_api.ts

```diff
@@ -0,0 +1,74 @@
+/*
```

Comment:
Lets make sure we have a follow up GitHub issue to merge `x-pack/plugins/triggers_actions_ui/public/application/lib/alert_api.ts` into this file. We can mark it as tech debt if not done in this PR.

Reply:
https://github.com/elastic/kibana/issues/60408

Done and filed under Code Debt